README.md

```diff
@@ -24,14 +24,18 @@ Typeshed supports Python versions 3.7 and up.
 
 ## Using
 
-If you're just using mypy (or pytype or PyCharm), as opposed to
+If you're just using a type-checker ([mypy](https://github.com/python/mypy/),
+[pyright](https://github.com/microsoft/pyright),
+[pytype](https://github.com/google/pytype/), PyCharm, ...), as opposed to
 developing it, you don't need to interact with the typeshed repo at
-all: a copy of standard library part of typeshed is bundled with mypy.
+all: a copy of standard library part of typeshed is bundled with mypy and pyright.
```

Comment:
Since in the context of "as a type checker user, do i have to care about this" the answer is still no
```suggestion
all: a copy of standard library part of typeshed is typically bundled with type checkers.
```